winlogtimeline/collector/parser.py

```diff
@@ -70,19 +73,22 @@ def parse_id_12(raw, record):
     record["description"] = "System Start"
 
     if len(data) == 7:
-        # print(data)
-        record["details"] = f'Starting Windows NT version {data[0]["#text"]}. {data[1]["#text"]}. {data[2]["#text"]} ' \
-                            f'at {data[6]["#text"][0:10]} {data[6]["#text"][11:23]} (UTC)'
+        time = get_string(data[6])
```

Comment:
The get_string not only fixes inconsistencies with records, but it also makes the code easier to read. Nice job.